util/dfu.py

```diff
@@ -27,26 +29,58 @@
             os.chdir(f"STM32/{args.board}")
             
             # If make has an error, raise an exception
-            subprocess.run(f"make", check=True)
-
-            if args.make:
-                exit(0)
-
-            # Open the port and put the device in DFU mode
-            if args.port:
-                brd = s.Serial(port=args.port)
-                try:
-                    while(brd.is_open):
-                        brd.write("DFU\n".encode('UTF-8'))
-                        sleep(0.1)
-                except s.SerialException:
-                    pass
-                finally:
-                    brd.close()
-
-                sleep(0.5)
-
-            subprocess.run(f"dfu-util -a 0 -D build/{args.board}.bin -s 0x08000000:leave", shell=True)
+            subprocess.run(f"make -j8", shell=True, check=True)
```

Comment:
We have seen issues on windows machines when compiling with the -j8 tag. Because the compilation is spread over different cores in some scenarios part of the project that is dependent on the githash.h is built before the githash.h is produced. This means the build will fail. 

Maybe it should be yet another option flag.

Reply:
Done